reference-architectures/standard/deploy-arch-ibm-pvs-inf-standard.md

```diff
@@ -0,0 +1,133 @@
+---
+copyright:
+  years: 2023
+lastupdated: "2023-02-21"
+
+keywords:
+
+subcollection: deployable-reference-architectures
+
+authors:
+  - name: Arnold Beilmann
+
+version: v7.0.0
+
+deployment-url: url #TODO
+
+docs: https://cloud.ibm.com/docs/solution-guide #TODO
+
+image_source: https://github.com/terraform-ibm-modules/terraform-ibm-powervs-infrastructure/reference-architectures/standard/deploy-arch-ibm-pvs-inf-standard.svg
+
+related_links:
+  - title: 'SAP in IBM Cloud documentation'
+    url: 'https://cloud.ibm.com/docs/sap'
+    description: 'SAP in IBM Cloud documentation.'
+  - title: 'Reference architecture for "Secure infrastructure on VPC for regulated industries"'
+    url: 'https://url' #TODO
+    description: 'Reference architecture for "Secure infrastructure on VPC for regulated industries"'
+
+use-case: ITServiceManagement
+
+industry: Technology
+
+compliance: SAPCertified
+
+content-type: reference-architecture
+
+---
+
+{{site.data.keyword.attribute-definition-list}}
+
+# Power infrastructure for deployable architectures - variation 'PowerVS workspace'
+{: #deploy-arch-ibm-pvs-inf-standard}
+{: toc-content-type="Power infrastructure for deployable architectures - variation 'PowerVS workspace'"}
+{: toc-industry="Technology"}
+{: toc-use-case="ITServiceManagement"}
+{: toc-compliance="SAPCertified"}
+
+Building upon the VPC service that were previously created when you deployed Secure infrastructure on VPC for regulated industries, Power Infrastructure for deployable architectures creates a Power Virtual Server workspace and connects it with VPC services. Proxy service for public internet access from PowerVS workspace is configured.
+
+Additional management components (NFS server, NTP forwarder, and DNS forwarder) may be installed on VPC optionally.
```

Comment:
```suggestion
You can optionally install more management components (such as NFS server, NTP forwarder, and DNS forwarder) on VPC.
```

Making it active voice and replacing a couple of words that we try not to use (additionally, may).